Documentation/security/launch-integrity/secure_launch_overview.rst

```diff
@@ -111,13 +111,22 @@ the mainline kernel where the TPM driver would be available.
 Basic Boot Flow
 ===============
 
+Outlined here is summary of the boot flow for Secure Launch. A more detailed
+review of Secure Launch process can be found in the Secure Launch
+Specification, a link is located in the `Resources`_ section.
+
 Pre-launch: *Phase where the environment is prepared and configured to initiate the
-secure launch in the GRUB bootloader.*
+secure launch by the boot chain.*
 
- - Prepare the CPU and the TPM for the launch.
+ - The SLRT is initialized and dl_stub is place in memory.
  - Load the kernel, initrd and ACM [2]_ into memory.
  - Setup the TXT heap and page tables describing the MLE [1]_ per the
    specification.
+ - If non-UEFI platform, dl_stub is called.
+ - If UEFI platforms, SLRT registered with UEFI and efi-stub called.
+ - Upon completion, the efi-stub calls dl_stub instead of EBS.
```

Comment:
These are not correct. efi-stub calls EBS before jumping to dlstub so this needs a little rewording.

Reply:
ack